.github/workflows/crucible-ci.yaml

```diff
@@ -19,16 +19,13 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
-    - name: ci-environment-info
-      uses: ./.github/actions/ci-environment-info
-
-    - name: Setup CI Environment
-      run: |
-        sudo ./setup-ci-environment
-
-    - name: Run fio CI
-      run: |
-        sudo ./run-ci --scenarios fio
+    - name: integration-tests
+      uses: ./.github/actions/integration-tests
+      with:
+        #scenarios: "fio"
```

Comment:
fio is the default scenario ?

Reply:
Yes.  The run-ci script has some defaults set so that it doesn't run forever if executed without any parameters.  At the moment I have this section setup to test the defaults, but ultimately this will be set explicitly.